src/types/Boolean.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { ThrowReporter } from 'io-ts/lib/ThrowReporter'
+import * as types from './index'
+
+describe('Boolean', () => {
+    it('should decode valid booleans', () => {
+        const dataProvider = ['false', 'true']
+
+        dataProvider.forEach(bool => {
+            const result = types.Boolean.decode(bool).value
+            expect(result).toEqual(bool)
+        })
+    })
+
+    it('should fail to decode invalid booleans', () => {
+        expect(() => {
+            const result = types.PortNumber.decode('0')
```

Comment:
`PortNumber` should be `Boolean` I guess (for all examples). Also There's a duplicate check for empty value.

Reply:
omg, totally missed that, thanks for pointing out